packages/client/src/users/putUser.js

```diff
@@ -1,14 +1,14 @@
-import client, { adaptError } from '../../helpers/client';
+import client, { adaptError } from '../helpers/client';
 import join from 'proper-url-join';
 
 /**
- * Method responsible for update user profile.
+ * Method responsible for update user's data.
  *
- * @function updateProfile
- * @memberof module:profile/client
+ * @function putUser
+ * @memberof module:users/client
  *
  * @param {number} id - User identifier.
- * @param {object} data - Profile data.
+ * @param {object} data - User data.
```

Comment:
Would be nice if you could also document which parameters this `data` object needs. What do you think?

Reply:
The JSDoc will be review in the Typescript phase 1 task, will add it then.